mlflow/store/_unity_catalog/registry_oss/rest_store_oss.py

```diff
@@ -1,19 +1,31 @@
 import functools
 
 from mlflow.protos.unity_catalog_oss_messages_pb2 import (
+    CreateModelVersion,
```

Comment:
I don't know that we need a `registry_oss` directory here.

Maybe we could rename the file to be `uc_oss_rest_store.py` and put it under registry with the existing `rest_store.py` that has the DB UC rest store.